cudf/dataframe.py

```diff
@@ -279,7 +282,21 @@ def assign(self, **kwargs):
 
     def head(self, n=5):
         """
-        Returns the first n rows as a new DataFrame
+        Return the first n rows.
+
+        This function returns the first n rows for the object based
+        on position. It is useful for quickly testing if your
```

Comment:
What do you mean by "based on position"? Isn't it sufficient to say "... returns the first n rows of the DataFrame."?